src/Logger.ts

```diff
@@ -0,0 +1,31 @@
+const ESC = "\u001b"
+const RESET = ESC + "[m"
+const GRAY  = ESC + "[37m"
+const WHITE = ESC + "[97m"
+const DARK  = ESC + "[90m"
+const RED   = ESC + "[91m"
+
+export default class Logger {
+
+	constructor(private readonly actor: string) {}
```

Comment:
Needs a color so we can diff what "actor" it is using colors, like you told me on discord.

Reply:
addressed by: [df3c529](https://github.com/creelonestudios/mailverse/pull/9/commits/df3c529f72307e04790a33966149ff4d42d04a79)